gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/internal/OrcaService.groovy

```diff
@@ -16,6 +16,7 @@
 
 package com.netflix.spinnaker.gate.services.internal
 
+import com.netflix.spinnaker.gate.security.RequestContext
```

Comment:
I think we should come up with a different interface for this. If we don't do a `withRequestContext` sort of thing, I'd probably just encourage a concrete, separate class that wraps `OrcaService` and we don't use that interface directly anymore. Some options, but I think this has some dragons that could lead to easy misuse.